src/views/FSL/lookup/lookup.vue

```diff
@@ -287,10 +287,11 @@
                     justify-content-center
                   "
                   >
-                  <span v-if="order.missMatchedPayments.length > 0" data-toggle="tooltip" data-placement="top" title="This has miss-matched payment info">
-                    <i class="fa fa-exclamation-circle text-danger mx-2" aria-hidden="true"></i>      
-                  </span>
+                  
                     {{ order.product.name | truncate(50) }}
+                    <span v-if="order.missMatchedPayments.length > 0" data-toggle="tooltip" data-placement="top" title="The account name doesn't match with credit check" class="badge badge-secondary pointer">
```

Comment:
Let's use `The debit account does not correspond to the account recorded on the bank statement.`